### PR TITLE
Fixing sort to work on all browsers

### PR DESCRIPTION
**Pull request by DXCanas**
## Summary

During testing, realized that sorting doesn't work for IE or Chrome, because their `Array.sort()` takes numbers as return values rather than booleans. Fixed in this patch.

Wasn't a trello card, but I wanted to fix it.

### CODE REVIEW

File: kolibri/plugins/management/assets/src/vue/user-page/index.vue [hunk 1]
```diff
@@ -234,6 +241,9 @@
     width: 100%
     word-break: break-all
 
+  th
```
Comment (rtibbles) [inline kolibri/plugins/management/assets/src/vue/user-page/index.vue:244]: This seems unrelated to the sort?

**Comment by rayykim** [comment]
Also fixed IE styling; headers were center aligned instead of left aligned.

**Comment by indirectlylit** [comment]
thank
